vyxal/elements.py

```diff
@@ -4058,11 +4049,7 @@ def vy_type(item, rhs=None, other=None, simple=False):
         )
     elif rhs is not None:
         return (vy_type(item, simple=simple), vy_type(rhs, simple=simple))
-    elif (
-        (x := type(item)) in (int, complex, float)
-        or is_sympy(item)
-        or isinstance(item, numpy.number)
-    ):
+    elif (x := type(item)) in (int, complex, float) or is_sympy(item):
```

Comment:
`type` not `vy_type`?

Reply:
I think because `vy_type` returns Vyxal-specific types? @Lyxal wrote this, not me

Reply:
Y'all realise that this is _inside_ the `vy_type` function right?